src/__experimental__/components/decimal/decimal.stories.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean, text, number, select } from '@storybook/addon-knobs';
+import { Decimal } from '.';
+import OptionsHelper from '../../../utils/helpers/options-helper';
+
+storiesOf('Experimental/Decimal', module)
+  .add('default', () => {
+    const align = select(
+      'align',
+      OptionsHelper.alignBinary,
+      Decimal.defaultProps.align
+    );
+    const precision = number('precision', Decimal.defaultProps.precision);
```

Comment:
is the precision prop working correctly? it looks like changing the knob in the storybook makes no difference

Reply:
It definitely wasn't, I've only finished writing it today. 😆